specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/customIpPrefix.json

```diff
@@ -0,0 +1,374 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/customIpPrefixes/{customIpPrefixName}": {
+      "delete": {
+        "tags": [
+          "CustomIPPrefixes"
+        ],
+        "operationId": "CustomIPPrefixes_Delete",
+        "description": "Deletes the specified custom IP prefix.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "customIpPrefixName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the CustomIpPrefix."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "200": {
+            "description": "Delete successful."
+          }
+        },
+        "x-ms-examples": {
+          "Delete custom IP prefix": {
+            "$ref": "./examples/CustomIpPrefixDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "get": {
+        "tags": [
+          "CustomIPPrefixes"
+        ],
+        "operationId": "CustomIPPrefixes_Get",
+        "description": "Gets the specified custom IP prefix in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "customIpPrefixName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the custom IP prefix."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
add default response

Reply:
Why do I need to add a default to just a few of the responses, but not all of them?

Reply:
Please add default response for all your operations.